src/extension/windows/windowsPlatform.ts

```diff
@@ -45,6 +45,9 @@ export class WindowsPlatform extends GeneralMobilePlatform {
 
             if (semver.gte(this.runOptions.reactNativeVersions.reactNativeWindowsVersion, "0.63.0")) {
                 this.runArguments.push("--logging");
+                if (enableDebug) {
```

Comment:
I think the issue was that specifying '--proxy' didn't really do anything - I fixed that and renamed to --remote-debugging.
So the code below in 'else' won't do anything - the if-else logic could be removed

Reply:
Removed `--proxy` and tested on 0.62 version and it seem to work fine. Thanks for pointing on this.